src/Entity/ProgrammaticFoundation/Measure.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace AppBundle\Entity\ProgrammaticFoundation;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="programmatic_foundation_measure")
+ * @UniqueEntity("slug", message="Le slug des mesures doit être unique")
+ * @UniqueEntity(
+ *     fields={"position", "approach"},
+ *     errorPath="position",
+ *     message="L'ordre d'affichage doit être unique pour un sous axe donné"
+ * )
```

Comment:
could you add for all new entity this class annotation: `@Algolia\Index(autoIndex=false)` pls?